ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -383,7 +515,6 @@ export default class ConfirmApproveContent extends Component {
               className="confirm-approve-content__address-identicon"
               diameter={20}
               address={toAddress}
-              image={tokenImage}
```

Comment:
we shouldn't use the `tokenImage` for the `identicon` of the address being granted approval.